lib/utils/buildHelper.js

```diff
@@ -0,0 +1,104 @@
+const log = require("@ui5/logger").getLogger("cli:commands:build");
+
+/**
+ * Creates a list of dependencies as String or RegExp by processing the given <code>dependencies</code> and
+ * <code>dependenciesRegExp</code> parameters. Dependencies are only added to the list if they exist in the
+ * project <code>tree</code>. Optionally, also the sub-dependencies of given <code>dependencies</code> are
+ * added to the list.
+ *
+ * @param {object} parameters Parameters
+ * @param {object} parameters.tree Project tree as generated by the
+ *   [@ui5/project.normalizer]{@link module:@ui5/project.normalizer}
+ * @param {Array.<string>} parameters.dependencies The dependencies to be considered
+ * @param {Array.<string>} [parameters.dependenciesRegExp] The dependencies to be considered as new
+ *   RegExp instances
+ * @param {boolean} [parameters.handleSubtree] Decides whether the sub-dependencies of the given
```

Comment:
```suggestion
 * @param {boolean} [parameters.handleSubtree=false] Decides whether the sub-dependencies of the given
```

I guess we can document the default behavior here.